src/elements/about-organizer-block.js

```diff
@@ -0,0 +1,92 @@
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/paper-button/paper-button.js';
+import '@polymer/marked-element/marked-element.js';
+import 'plastic-image/plastic-image.js';
+import { ReduxMixin } from '../mixins/redux-mixin.js';
+import './hoverboard-icons.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+class AboutOrganizerBlock extends ReduxMixin(PolymerElement) {
+  static get template() {
+    return html`
+    <style is="custom-style" include="shared-styles flex flex-alignment positioning"></style>
+
+    <style>
+      :host {
+        display: block;
+      }
+
+      .block:not(:last-of-type) {
+        margin-bottom: 32px;
+      }
+
+      .team-icon {
+        --iron-icon-height: 160px;
+        --iron-icon-width: 160px;
+        --iron-icon-fill-color: var(--default-primary-color);
+        max-width: 50%;
+      }
+
+      .image-link {
+        width: 80%;
+        height: 80%;
+      }
+
+      .organizers-photo {
+        width: 100%;
+        height: 100%;
+      }
+
+      .description {
+        color: var(--secondary-text-color);
+      }
+
+      paper-button {
+        margin: 0;
+      }
+    </style>
+
+    <div class="container" layout="" horizontal="">
+      <div layout="" horizontal="" center-center="" flex="" hidden\$="[[viewport.isPhone]]">
```

Comment:
```suggestion
      <div layout horizontal center-center flex hidden$="[[viewport.isPhone]]">
```
Lets make attributes clean as much as possible